infra/gcp/clusters/projects/kubernetes-public/aaa/12-iam.tf

```diff
@@ -0,0 +1,40 @@
+/*
+Ensure google groups defined in 'members' have read-only access to
+all monitoring data and configurations.
+Detailed list of permissions: https://cloud.google.com/monitoring/access-control#roles
+*/
+resource "google_project_iam_binding" "readonlymonitoringbinding" {
+  project = data.google_project.project.id
+  role    = "roles/monitoring.viewer"
+
+  members = [
+    "group:k8s-infra-rbac-gcsweb@kubernetes.io",
+    "group:k8s-infra-rbac-k8s-io-canary@kubernetes.io",
+    "group:k8s-infra-rbac-k8s-io-prod@kubernetes.io",
+    "group:k8s-infra-rbac-perfdash@kubernetes.io",
+    "group:k8s-infra-rbac-publishing-bot@kubernetes.io",
+    "group:k8s-infra-rbac-triageparty-release@kubernetes.io",
+    "group:k8s-infra-rbac-slack-infra@kubernetes.io",
+    "group:k8s-infra-rbac-node-perf-dash@kubernetes.io",
```

Comment:
Would it make sense to use `gke-security-groups@kubernetes.io` instead?  Thinking it'd be nice to avoid duplicating this list

Reply:
Totally make sense to use `gke-security-groups@kubernetes.io`. I'll update the PR.